app/del_replica.go

```diff
@@ -0,0 +1,38 @@
+package app
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+func DelReplicaCmd() cli.Command {
+	return cli.Command{
+		Name:      "del-replica",
+		ShortName: "del",
+		Action: func(c *cli.Context) {
+			if err := deleteReplica(c); err != nil {
+				logrus.Fatalf("Error running del replica command: %v", err)
+			}
+		},
+	}
+}
+
+func deleteReplica(c *cli.Context) error {
+	controllerClient := getCli(c)
+	out, err := controllerClient.DeleteVolume()
```

Comment:
Delete replica vs. delete volume? Which is it?

Reply:
it's delete volume. Will change that.